DataFormats/TrackerCommon/interface/LegacyTrackerTopology.h

```diff
@@ -0,0 +1,97 @@
+#ifndef TRACKER_TOPOLOGY_LEGACY_H
+#define TRACKER_TOPOLOGY_LEGACY_H
+
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "DataFormats/DetId/interface/DetId.h"
+
+
+// WARNING: this header has been introduced to call a TrackerTopology
+// object whenever it is not possible to access it through an EventSetup.
+// Do not use it if that is the case.
+
+
+namespace LegacyTrackerTopology {
+  
+  inline static std::unique_ptr<TrackerTopology> getTrackerTopology()
```

Comment:
Hi @alesaggio, 
I think there might be a bit of ambiguity on what "Legacy" actually means in this context for what concern the Pixel part. Your implementation here is for Phase-0 Pixel Tracker only - I think it should be written somewhere as a disclaimer or allow the possibility to construct the Topology in this way also for Phase-I detector (since Phase0 Strips overlap in time with Phase-I Pixel, i.e. the current detector)